includes/specials/SpecialListfiles.php

```diff
@@ -74,7 +74,14 @@ function __construct( IContextSource $context, $userName = null, $search = '', $
 			$nt = Title::newFromText( $userName, NS_USER );
 			if ( !is_null( $nt ) ) {
 				$this->mUserName = $nt->getText();
-				$this->mQueryConds['img_user_text'] = $this->mUserName;
+				/* Wikia change begin */
+				$userId = User::idFromName( $this->mUserName );
+				if ( !empty( $userId ) ) {
+					$this->mQueryConds['img_user'] = $userId;
+				} else {
+					$this->mQueryConds['img_user_text'] = $this->mUserName;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "img_user_text" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)